javascript.js

```diff
@@ -8,16 +8,24 @@ const pitcherConfig = require('./pitcher.config')
 
 module.exports = {
   extends: ['eslint:recommended'],
-  plugins: ['sort-imports-es6-autofix'],
+  plugins: ['import', 'node'],
   rules: {
-    'sort-imports-es6-autofix/sort-imports-es6': [
+    'import/order': [
       'error',
       {
-        ignoreCase: false,
-        ignoreMemberSort: false,
-        memberSyntaxSortOrder: ['all', 'single', 'multiple', 'none'],
+        groups: ['builtin', 'external', 'internal', 'parent', 'sibling', 'index', 'object', 'type'],
+        pathGroups: [
+          {
+            pattern: '@/**',
+            group: 'external',
+            position: 'after',
+          },
+        ],
+        pathGroupsExcludedImportTypes: ['builtin'],
       },
     ],
+    'import/newline-after-import': ['error', { count: 1 }],
+    'node/file-extension-in-import': ['error', 'always', { '.js': 'never', '.vue': 'never', '.ts': 'never' }],
```

Comment:
Has this been tested with Vite? I thought Vite doesn't allow extension-less Vue imports.
https://github.com/vitejs/vite/issues/178

Reply:
We stopped using Vite for now because things like this one. I think we will revisit when it is more mature :)